deepchem/rl/a2c.py

```diff
@@ -40,17 +39,28 @@ def __call__(self, outputs, labels, weights):
 
 
 class A2CLossContinuous(object):
-  """This class computes the loss function for A2C with continuous action spaces."""
+  """This class computes the loss function for A2C with continuous action spaces.
+
+  Note
+  ----
+  This class requires tensorflow-probability to be installed.
```

Comment:
This is an internal class that users don't use directly, so documenting this here isn't useful.  Instead, add to the documentation for A2C that continuous action spaces require tfp.  (It also would probably be very easy to get rid of this dependency, since the only thing it's being used for is to compute a normal probability distribution.)

Reply:
Good suggestion! I've added in a note there. I think it's harmless to leave the dependency since we might end also up using tfp for the normalizing flows API that @ncfrey is working on